pkgs/development/python-modules/pytest-helpers-namespace/default.nix

```diff
@@ -0,0 +1,30 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, pytest_3
```

Comment:
I just merged a changeset removing `pytest_3`. If it's not possible to use the latest `pytest` we have, I suggest just disabling the tests.

Reply:
I disabled them